docs/topics/cli-run.adoc

```diff
@@ -38,7 +38,9 @@ The following command analyzes the `com.acme` and `org.apache` packages of the l
 
 [source,terminal,subs="attributes+"]
 ----
-$ <{ProductShortName}_HOME>/{mta-cli} \
+
+$ <{ProductShortName}_HOME>/bin/windup-cli \
```

Comment:
The removed line was the correct one

Reply:
For MTR only

For MTA changes see `mta-cli-run.adoc`
```
windup-documentation/docs/cli-guide-mtr/master.adoc:
   32  
   33  // Run the CLI
   34: include::topics/cli-run.adoc[leveloffset=+2]
   35  
   36  // Use OpenRewrite recipes

windup-documentation/docs/cli-guide/master.adoc:
   36  
   37  // Run the CLI
   38: include::topics/mta-cli-run.adoc[leveloffset=+2]